README.rst

```diff
@@ -106,6 +106,5 @@ The easiest way to learn what files should look like is to read the ``rst`` file
 A short list of style guidance:
 
 * Headings should be in sentance case
-* If using ``rst`` tables, use "simple" tables; "grid" tables
-(markup with ``+`` in corners) generate HTML with accessability problems.
+* If using ``rst`` tables, use "simple" tables; "grid" tables (markup with ``+`` in corners) generate HTML with accessability problems.
```

Comment:
The word "accessability" is mispelled. It should be "accessibility".

Reply:
Done.